tests/Qdrant.Client.Tests/PointTests.cs

```diff
@@ -305,6 +305,64 @@ public async Task Count_with_filter()
 		Assert.Equal(1ul, count);
 	}
 
+	[Fact]
+	public async Task UpdateBatch()
+	{
+		await CreateAndSeedCollection("collection_1");
+
+		var operations = new PointsUpdateOperation[]
+		{
+			new()
+			{
+				Upsert = new()
+				{
+					Points =
+					{
+						new PointStruct[]
+						{
+							new() { Id = 1, Vectors = new float[] { 0.3f, 1.5f } }
+						}
+					}
+				}
+			},
+			new()
+			{
+				UpdateVectors = new()
+				{
+					Points =
+					{
+						new PointVectors[]
+						{
+							new() { Id = 1, Vectors = new float[] { 10.5f, 11.5f } }
+						}
+					}
+				}
+			}
+		};
+
+		var response = await _client.UpdateBatchAsync("collection_1", operations);
+
+		Assert.All(response, r => r.Status.Should().Be(UpdateStatus.Completed));
+	}
+
+	[Fact]
+	public async Task Recommend_with_vector()
+	{
+		await CreateAndSeedCollection("collection_1");
+
+		var points = await _client.RecommendAsync(
+			"collection_1",
+			positive: new PointId[] { 8 },
+			positiveVectors: new Vector[] {
+				new float[] { 3.5f, 4.5f }
+			},
+			limit: 1
+		);
+
+		var point = Assert.Single(points);
+		Assert.Equal(9ul, point.Id);
+	}
```

Comment:
Move this test method next to the other Recommend tests

Reply:
https://github.com/qdrant/qdrant-dotnet/pull/42/commits/30f944cd5557184d9520cc129d175e7deb94976c